crates/evm/src/fuzz/invariant/executor.rs

```diff
@@ -622,48 +606,31 @@ fn can_continue(
 
             failures.revert_reason = Some(error.revert_reason.clone());
 
-            // Hacky to provide the full error to the user.
-            for invariant in invariant_contract.invariant_functions.iter() {
-                failures.failed_invariants.insert(
-                    invariant.name.clone(),
-                    (Some(error.clone()), invariant.to_owned().clone()),
-                );
-            }
```

Comment:
Why was this removed?

Reply:
Can you take a look at #5726 pls? I think a bug was introduced but I can't manage to find when or where